src/content/en/2020/performance.md

```diff
@@ -1,23 +1,338 @@
+
+In this chapter, we will be looking at real-world performance data provided by the [Chrome User Experience Report (CrUX)](https://developers.google.com/web/tools/chrome-user-experience-report) through the lens of those new developments as well as analyzing a handful of other relevant metrics. It’s important to note that due to the limitations of iOS, CrUX mobile results don’t include devices running Apple’s mobile operating system. This fact will undeniably affect our analysis, especially when examining metric performance on a per-country basis.
+
+Let’s dive in.
+
+### Lighthouse Performance Score
+
+In May 2020, [Lighthouse 6 was released](https://github.com/GoogleChrome/lighthouse/releases/tag/v6.0.0). The new major version of the popular performance auditing suite introduced notable changes to its Performance Score algorithm. Performance Score is a high-level portrayal of site speed. **In Lighthouse 6, the score is measured with six—not five—metrics**. First Meaningful Paint and First CPU Idle were removed and replaced with Largest Contentful Paint (LCP), Total Blocking Time (TBT, the lab equivalent of First Input Delay) and Cumulative Layout Shift (CLS).
+
+The new scoring algorithm is prioritizing the new generation of performance metrics: Core Web Vitals and deprioritizing First Contentful Paint (FCP), Time to Interactive (TTI) and Speed Index, as their score weight decreases. The algorithm also now distinctly emphasizes three aspects of user experience: **interactivity** (Total Blocking Time and Time to Interactive), **visual stability** (Cumulative Layout Shift) and **perceived loading** (First Contentful Paint, Speed Index, Largest Contentful Paint).
+
+Additionally, **the score is now calculated using different reference points for desktop and mobile**. What this means in practice is that it will be less forgiving on desktop (expecting fast websites) and more relaxed on mobile (since benchmark performance on mobile is less quick than desktop). You can compare your Lighthouse 5 and 6 score difference in [the Lighthouse Score calculator](https://googlechrome.github.io/lighthouse/scorecalc/). So, how did the scores really change?
+
+{{ figure_markup(
+  image="performance-change-in-lighthouse-score.png",
+  caption="Difference in Lighthouse Performance Score between versions 5 and 6.",
+  description="Column chart showcasing the change in Performance Score between versions 5 and 6. The highest percentage of websites (4%) observed no changes to the score, and the number of sites with score decreases outweighs score improvements.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vT3ukFCTxx__dTBLwDQ3K_hCtXdLRGOGUYdR_TO043n_aCTdcwkxPUku9Qfoal6BESiV5RMtd0nEbqT/pubchart?oid=786955541&format=interactive"
+  )
+}}
```

Comment:
We need to add `sheets_gid` and `sql_file` parameters to all the figures:

```suggestion
{{ figure_markup(
  image="performance-change-in-lighthouse-score.png",
  caption="Difference in Lighthouse Performance Score between versions 5 and 6.",
  description="Column chart showcasing the change in Performance Score between versions 5 and 6. The highest percentage of websites (4%) observed no changes to the score, and the number of sites with score decreases outweighs score improvements.",
  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vT3ukFCTxx__dTBLwDQ3K_hCtXdLRGOGUYdR_TO043n_aCTdcwkxPUku9Qfoal6BESiV5RMtd0nEbqT/pubchart?oid=786955541&format=interactive",
  sheets_gid="518150031",
  sql_file="lh6_vs_lh5_performance_score_distribution.sql"
  )
}}
```

Reply:
@bazzadp done 👍 hope I didn't mismatch anything there 😄